### DISCUSSION TITLE
How to use any local or my own voice for text to speech

### DISCUSSION THREAD

**Discussion by souvikg544** [coqui-ai/TTS#1892] [General Q&A]
I saw the area in the github where it tells about cloning your own voice.But I am not quite sure how to use config.json and get your own vocoder models

**Comment by thorstenMueller** [comment]
Training your own TTS/vocoder model is a not too simple and time consuming task. But if you would like to do so this doc (https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html) might help. Or if you prefer video i've made a technical walkthrough on how to train your own TTS model on my Youtube channel (https://youtu.be/4YT8WZT_x48).
